simplq/src/components/pages/Join/index.jsx

```diff
@@ -39,4 +42,31 @@ export function JoinQueue(props) {
   );
 }
 
-export default JoinQueue;
+export function JoinQueueWithLink(props) {
+  const [queueLink, setQueueLink] = useState('');
+
+  const handleClick = (link) => {
+    const queueId = link.split('/').pop();
+    props.history.push(`/j/${queueId}`);
+  };
+
+  return (
+    <>
+      <Banner />
+      <InputField
+        placeholder="Enter queue link"
+        onKeyPress={(e) => handleEnterPress(e, () => handleClick(queueLink))}
+        value={queueLink}
+        onChange={(e) => setQueueLink(e.target.value)}
+        className={styles.input}
+        // error={invalidName}
```

Comment:
Can we remove these comments?

Reply:
I'm keeping it there since we haven't really handled the errors while pasting a link. Ideally, I want this to work with just the name as well the whole link, so we'd need a validator specific to it later on.